### ISSUE TITLE
Hyphens get removed from output

### ISSUE THREAD

**Issue by alexgleason** [ianstormtaylor/to-title-case#7]
Example:

```
> var toTitleCase = require('to-title-case')
undefined
> toTitleCase('hello world')
'Hello World'
> toTitleCase('hello-world')
'Hello World'
```

Hyphens in the input should be retained in the output.

**Comment by ianstormtaylor** [comment]
This is by design, at least as far as these inter-related case helpers are concerned. Because if you pass in something like `'a-slug-cased-string'`, it should convert it away from slug case. But if you're looking for a purely title-case-handling solution, it's going to need to be different unfortunately.

**Comment by alexgleason** [comment]
I see, so you are considering "slug case" and title case to be on the same level.

In that case I agree with the solution, but I'm not sure I agree with the premise. A phrase containing hyphens *could* be a slug, but it could also be a hyphenated English phrase.

wrought-iron should become "Wrought-Iron" but my-first-blog-post should become "My First Blog Post." Yet these are each solving two different problems. I would personally rather make the least assumptions possible about the input text since it's easy to chain additionally functionality but not to remove it.

**Comment by ianstormtaylor** [comment]
Yup, I agree with you. But this set of libraries (really should be in a single module these days with lodash-style imports) are focused on conversion between cases, and not prioritizing any specific one. So unfortunately `wrought-iron` will be recognized as slug case, and converted.
